strapi/strapi
# Splitting the profile page

### What does it do?

- Create a reusable select component in strapi-helper-plugins
- Moved the language dropdown in its own dedicated section in ProfilePage
- Extracting the Password section in its own one
- Adding titles on the "Experience" section and on the "Password" section

### How to test?

- Log in the application
- You should not see the language switcher anymore on the top right side of the screen
- Click on your name on the top right side of the screen
- Select "profile"
- You should now see the new profile page
